SPECS/rapidjson/CVE-2024-39684.nopatch

```diff
@@ -0,0 +1 @@
+CVE-2024-39684 is a duplicate of CVE-2024-38517
```

Comment:
I think apart from the `kernel` spec we generally dispute the CVEs instead of adding a no-patch file.

Reply:
Should we update the wiki section ***Resolve and Dispute Process*** since nopatch files is listed as one mechanism of dispute.

Reply:
I think we should. That's a very good idea.
Kudos for the initiative:)